Packages/MIES/MIES_Utilities.ipf

```diff
@@ -3116,6 +3116,26 @@ threadsafe Function/WAVE ListToTextWaveMD(list, dims, [rowSep, colSep, laySep, c
 	return output
 End
 
+#if IgorVersion() < 9.0
+
+threadsafe Function/S ReplicateString(string str, variable numTotalCopies)
+
+	variable i
+	string list = ""
+
+	if(!IsFinite(numTotalCopies) || numTotalCopies <= 0)
+		return str
+	endif
+
+	for(i = 0; i < numTotalCopies; i += 1)
```

Comment:
Just a minor thought:
I have not benched it, though I would have used a two liner with
PadString + ReplaceString here.

Reply:
Might be worth a thought. But IP8 is also on its way out so I'd leave it as it is.